### DISCUSSION TITLE
Application API: Create User API keys

### DISCUSSION THREAD

**Discussion by HerrSammyDE** [pelican-dev/panel#530] [Ideas]
Hi,

It would be great if you could create API keys for users via the Application API.

So far we always had to implement the https://github.com/rene-roscher/pterodactyl-api-addon to create API keys for users

**Comment by HerrSammyDE** [comment]
Someone has already rewritten it (untested) for Pelican, but a direct integration would be better! Thanks @wisingman

https://github.com/wisingman/pelican-api-addon

**Reply by wisingman** [reply]
I was able to get it to work after reading up on implementing VCS repos with composer, and will post some instructions in the near future. Any contributions would be appreciated as this is not my area of expertise ! Use with caution !
